<launch>
  <param name="robot_description" command="cat $(find hebiros_description)/urdf/x_demo.urdf"/>

  <node name="hebiros_node" pkg="hebiros" type="hebiros_node" respawn="false" output="screen"/>

  <node name="example_gazebo_node" pkg="hebiros_advanced_examples" type="example_gazebo_node"/>
</launch>
